### PR TITLE
Ansible-playbook output fixes

### PR DESCRIPTION
**Pull request by vrutkovs**
Addresses https://github.com/openshift/release/pull/1026/files#r203040353
and https://github.com/openshift/release/pull/1026/files#r203040555

### CODE REVIEW

File: cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml [hunk 1]
```diff
@@ -170,7 +170,7 @@ objects:
         #!/bin/bash
         set -euo pipefail
 
-        trap 'rc=$?; if [[ $rc -ne 0 ]]; then touch /tmp/exit; fi; exit $rc' EXIT
+        trap 'rc=$?; if [[ $rc -ne 0 ]]; then touch /tmp/shared/exit; fi; exit $rc' EXIT
```
[the remark this answers is not in the document]
Comment (vrutkovs) [reply cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml]: It already has `mountPath: /tmp/shared`

File: cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml [hunk 2]
```diff
@@ -153,7 +153,7 @@ objects:
       image: ${IMAGE_INSTALLER}
       volumeMounts:
       - name: shared-tmp
-        mountPath: /tmp
+        mountPath: /tmp/shared
```
[the remark this answers is not in the document]
Comment (vrutkovs) [reply cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml]: Other parts are already using `/tmp/shared`.

`/tmp/output` in test container and `/tmp/cluster` don't need to be shared

File: cluster/ci/config/prow/jobs/cluster-launch-src.yaml [hunk 3]
```diff
@@ -165,7 +165,7 @@ objects:
 
         ansible-playbook -vv \
           -e openshift_test_repo=${RPM_REPO} -e 'oreg_url=${IMAGE_FORMAT}' \
-          playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
```
[the remark this answers is not in the document]
Comment (vrutkovs) [reply cluster/ci/config/prow/jobs/cluster-launch-src.yaml]: Created https://github.com/openshift/release/pull/1075

File: cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (smarterclayton) [reply cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml]: Yes, it does.  See teardown.

[the remark this answers is not in the document]
Comment (vrutkovs) [reply cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml]: Ah, yes. Updated

**Comment by vrutkovs** [comment]
We don't need that anymore, #1077 should be enough

**Pull request closed without merging**